.github/workflows/build-website.yml

```diff
@@ -7,6 +7,23 @@ on:
     branches: [ main ]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout 🛎️
+      uses: actions/checkout@master
+      
+    - name: Setup Node 🔧
+      uses: actions/setup-node@v3
+      with:
+        node-version: '14'
+
+    - name: Run lint
+      run: |
+        yarn
+        yarn eslint --config .eslintrc.js --ignore-path .eslintignore .
```

Comment:
Could consider adding this to the `Makefile` so you can `make lint`. I don't feel strongly and don't need to review again regardless of your decision! :)